### ISSUE TITLE
Raptoreum

### ISSUE THREAD

**Issue by acos0874** [RainbowMiner/RainbowMiner#1725]
Sorry in advance if this is already in RBM and I've missed it. Please can it be added with support for zergpool?

**Comment by RainbowMiner** [comment]
Yes, we do already have Ghostrider - the algorithm is called Take2, the coins are RTM and WATC

**Comment by acos0874** [comment]
OK thanks!

**Comment by acos0874** [comment]
Hi, I'm having trouble getting this running. I have CPU mining enabled on a ryzen 3800x and RTM enabled on zergpoolcoins. I have all miners enabled except for a list of exclusions

`ttminer,ccminer,CcminerAlexis78,CcminerBmw512,ethminer,CcminerLyra2v3,CcminerLyra2z330,CcminerSha3d,CcminerTpruvot,CcminerVerus,Ewbf,Nsfminer,ccminermtp,ccminermtp10,ccminertpruvot,teamblack,cryptodredge25,wildrig,miniz,lolminer,progpowznvidia,nbminer,nanominer,spminer,xmrig,ccminerxevan,firopownvidia,verthash`

Any ideas why it won't show up?

[debug_2021-10-31.zip](https://github.com/RainbowMiner/RainbowMiner/files/7449450/debug_2021-10-31.zip)
